src/main/java/gift/auth/security/JwtFilter.java

```diff
@@ -0,0 +1,63 @@
+package gift.auth.security;
+
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+@Component
+public class JwtFilter extends OncePerRequestFilter  {
+
+    private final JwtUtil jwtUtil;
+
+    public JwtFilter(JwtUtil jwtUtil) {
+        this.jwtUtil = jwtUtil;
+    }
+
+    @Override
+    protected boolean shouldNotFilter(HttpServletRequest request) throws ServletException {
+        String[] excludePath = {"/members/register", "/members/login"};
+        String path = request.getRequestURI();
+        return Arrays
+                .stream(excludePath)
+                .anyMatch(path::startsWith);
+    }
+
+    @Override
+    protected void doFilterInternal(
+            HttpServletRequest request,
+            HttpServletResponse response,
+            FilterChain filterChain) throws ServletException, IOException {
+
+        String authHeader = request.getHeader("Authorization");
+
+        if (authHeader == null) {
+            response.sendError(HttpStatus.UNAUTHORIZED.value(), "인증이 유효하지 않습니다.");
+            return;
+        }
+
+        if (!authHeader.startsWith("Bearer ")) {
+            response.sendError(HttpStatus.FORBIDDEN.value(), "인증에 실패하였습니다.");
+            return;
+        }
+
+        String token = authHeader.substring(7);
```

Comment:
7이라는 숫자가 Bearer 를 제외한 의미를 가질 수 있게 상수로 정의하는 것은 어떨지 궁금해요💭

```suggestion
        private static final String BEARER_PREFIX = "Bearer ";
        private static final int BEARER_PREFIX_LENGTH = BEARER_PREFIX.length();
        
        String token = authHeader.substring(BEARER_PREFIX_LENGTH);
```

Reply:
값을 상수로 정의해서 의미 부여하는 게 확실히 가독성을 높이는 것 같아 좋은 방법이라 생각합니다!
코드를 싹 훑어보면서 리팩토링해보겠습니다.